habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1682,6 +1872,48 @@ def action_space(self):
         return EmptySpace()
 
 
+@registry.register_task_action
+class LookUpDownContinuousAction(WaypointAction):
+    name: str = "look_up_down_continuous"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._turn_angle = self._config.turn_angle
+        self._max_wait_duration = self._config.max_wait_duration
+
+    def step(self, look_up_down, *args: Any, **kwargs: Any):
+        r"""Update ``_metric``, this method is called from ``Env`` on each
+        ``step``.
+        """
+        xyt_waypoint = np.array([0.0, 0.0, 0.0])
+        if look_up_down > 0:
+            camera_delta_angle = self._turn_angle
+        elif look_up_down < 0:
+            camera_delta_angle = -self._turn_angle
+        else:
+            camera_delta_angle = 0.0
```

Comment:
Major concern: It seems like it's quite hard for a continuous policy to command the camera to stay in place (which seems like a common thing), as it would need to output exactly zero.

Reply:
maybe we add a small threshold value? `look_up_down > threshold`?

Reply:
As I mentioned here

> 1. Why are camera pitch actions delta angles, instead of directly commanding angles? We can simply map [-1, 1] to [min_pitch, max_pitch]. Maybe I don't have context on this, @ykarmesh ?

Maybe it's easier to just control the absolute camera angle?

Reply:
In reinforcement learning, it is better to use delta angles, rather than directly commanding the angles. Thank you!

Reply:
Agree with Jimmy here

Reply:
Done, I will update the PR. thank you!